python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -140,7 +140,13 @@ def fit(self, y: cudf.Series) -> "LabelEncoder":
 
         y = _enforce_str(y)
 
-        self._cats = nvcategory.from_strings(y.data)
+        nvs = y._data[list(y._data.keys())[0]]._nvstrings
```

Comment:
Weird... future step - maybe a wrapper for this? Seems like an awkward way to get _nvstrings each time

Reply:
`_nvstrings` will be coing away when cuDF Python starts using cudf::columns if I'm not mistaken so changing this to a wrapper seems like a great idea to prepare for that indeed :)